app/services/message_employee_matcher.rb

```diff
@@ -36,17 +41,13 @@ def day_count
     message.days_after_start
   end
 
-  def time_to_send_message?(time_zone)
-    employee_current_time = Time.current.in_time_zone(time_zone)
-
-    if employee_current_time.day == Time.current.day
-      employee_current_time_value = employee_current_time.strftime("%H%M").to_i
-      message_time_value = message.time_of_day.strftime("%H%M").to_i
-
-      employee_current_time_value >= message_time_value
-    else
-      false
-    end
+  def time_to_send_message?(employee)
+    employee_start_date = day_count.business_days.after(employee.started_on)
```

Comment:
I am not sure what we have here is an employee start date (unless I am misunderstanding the code here, which is totally possible)...is there a better var name we can use than ` employee_start_date`